packages-web/datagrid/src/components/__tests__/Pagination.spec.tsx

```diff
@@ -1,36 +1,60 @@
 import { shallow } from "enzyme";
 import { createElement } from "react";
-import { ClientSidePagination, ClientSidePaginationProps, Pagination, PaginationProps } from "../Pagination";
+import { Pagination, PaginationProps } from "../Pagination";
 
 describe("Pagination", () => {
     it("renders the structure correctly", () => {
         const component = shallow(<Pagination {...mockPaginationProps()} />);
 
         expect(component).toMatchSnapshot();
     });
-});
 
-describe("Client Side Pagination", () => {
-    it("renders the structure correctly", () => {
-        const component = shallow(<ClientSidePagination {...mockClientSideProps()} />);
+    it("disables previous button if is first page", () => {
+        const component = shallow(<Pagination {...mockPaginationProps()} canPreviousPage={false} />);
 
-        expect(component).toMatchSnapshot();
+        expect(
+            component
+                .find(".btn")
+                .at(1)
+                .prop("disabled")
+        ).toBeTruthy();
+    });
+
+    it("disables first page button if is first page", () => {
```

Comment:
There are no committed snapshots

Reply:
Oh.. i did proper tests, but i can also expects to match snapshot in this case

Reply:
Done